all_test.go

```diff
@@ -1300,8 +1300,8 @@ func TestAuthUpdateIdentity(t *testing.T) {
 		ModifiedBy:      newModifiedBy,
 		Type:            UserTypeLDAP,
 	}
-	if err := c.UpdateIdentity(&notFoundUser); err != ErrIdentityAuthNotFound {
-		t.Fatalf("TestUpdateIdentity failed: Expected ErrIdentityAuthNotFound, but got: %v", err)
+	if err := c.UpdateIdentity(&notFoundUser); err == nil {
+		t.Fatalf("Expected ErrIdentityAuthNotFound, but got: %v", err)
```

Comment:
wont the err in %v be nil always? shouldn't it be !errors.Is(err, ErrIdentityAuthNotFound)?